### PR TITLE
Update to Flow 0.34

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
this got screwed up...

File: js/noms/src/type.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import search from './binary-search.js';
 import {staticTypeCache} from './type-cache.js';
 
 export interface TypeDesc {
-  kind: NomsKind;
```
Comment (kalman) [inline js/noms/src/type.js]: whoa what does this mean

**Comment by arv** [comment]
Fixed.

File: js/noms/src/type.js (hunk 1 above)
Comment (arv) [inline js/noms/src/type.js]: https://flowtype.org/blog/2016/10/04/Property-Variance.html

**Comment by arv** [comment]
jenkins: test this

**Pull request merged**